File: app/src/main/java/com/example/logintest/cicloone.java
package com.example.logintest;

import android.os.Bundle;
import androidx.appcompat.app.AppCompatActivity;
import androidx.viewpager2.widget.ViewPager2;

public class cicloone extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activityone);

        // Obtener la referencia del ViewPager2
        ViewPager2 viewPager = findViewById(R.id.viewPager);

        // Crear instancia del adaptador
        imashadap imageAdapter = new imashadap();

        // Establecer el adaptador
        viewPager.setAdapter(imageAdapter);
    }
}

File: app/src/main/java/com/example/logintest/cicloone.kt
package com.example.logintest

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import androidx.viewpager2.widget.ViewPager2

class cicloone : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activityone)

        // Obtener la referencia del ViewPager2
        val viewPager = findViewById<ViewPager2>(R.id.viewPager)

        // Crear instancia del adaptador
        val imageAdapter = imashadap()

        // Establecer el adaptador
        viewPager.adapter = imageAdapter
    }
}
